### DISCUSSION TITLE
drop events from specific domains

### DISCUSSION THREAD

**Discussion by OldhamMade** [plausible/analytics#1654] [Anything Else]
It would be very useful to be able to configure a list of domains somewhere in the settings for which events are ignored/dropped.

For example, for development our team uses `our-app.local` instead of `localhost`, and we have `staging.our-app.com` and other "review" domains we also use.

It would be great if we could set a couple of patterns in the plausible settings screen for a website to ignore events from these domains.

**Comment by metmarkosaric** [comment]
thanks for the feedback @OldhamMade!

currently in order to exclude yourself or your team from being counted, we have this localstorage flag feature - have you seen it? https://plausible.io/docs/excluding-localstorage

**Comment by OldhamMade** [comment]
Interesting! It's a useful option, but I think having something configurable in the server is a good backstop in case someone forgets to set the local storage flag.
